cfgov/jinja2/v1/_includes/molecules/related-metadata.html

```diff
@@ -0,0 +1,109 @@
+{# ==========================================================================
+
+   related_metadata.render()
+
+   ==========================================================================
+
+   Description:
+
+   Creates related metadata markup when given:
+
+   is_half_width:         A Boolean indicating whether the list items should be at half
+                          width. Defaults to False.
+
+   related_metadata_list: A list of relatated metadata items.
+
+   ========================================================================== #}
+
+{% macro render(related_metadata_list, is_half_width=false) %}
+{% set types_lookup = {
+    'date': _date,
+    'list': _list,
+    'text': _text
+} %}
+<div class="m-related-metadata
+            {{ 'm-related-metadata__half-width' if is_half_width else '' }}">
+    <h2 class="header-slug">
+        <span class="header-slug_inner">
+            Related Metadata
+        </span>
+    </h2>
+    {% for list_item in related_metadata_list %}
+    <div class="m-related-metadata_item-container">
+        <h4>
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/5db4db6d035790fefeaa840c4830e2f7d9629292